### PR TITLE
[15.0][MIG] account_invoice_report_grouped_by_picking_sale_mrp: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT37268

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @chienandalu @pedrobaeza

**Comment by pedrobaeza** [comment]
/ocabot migration account_invoice_report_grouped_by_picking_sale_mrp

File: account_invoice_report_grouped_by_picking_sale_mrp/tests/test_report_grouped_sale_mrp.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form, TransactionCase
+
+
+class TestReportGroupedSaleMrp(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestReportGroupedSaleMrp, cls).setUpClass()
```
Comment (pedrobaeza) [inline account_invoice_report_grouped_by_picking_sale_mrp/tests/test_report_grouped_sale_mrp.py]:
```suggestion
        super().setUpClass()
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza change done

**Review by chienandalu — APPROVED** [review]

**Pull request merged**